maven-plugins/enforcer-maven-plugin/src/test/java/io/helidon/build/maven/enforcer/ExcludeTest.java

```diff
@@ -98,4 +108,369 @@ void testContains() {
         excluded = exclude.matches(FileRequest.create("src/resources/sbin/copyright.txt"));
         assertThat("Should not matches directory", excluded, is(false));
     }
+
+    @Test
```

Comment:
Let's have separate test class called `GitIgnoreTest`.
Also let do "blackbox" testing, and invoke the main API, you can overload the API to pass a list of lines instead of a Path.